components/dashboard/src/settings/Preferences.tsx

```diff
@@ -153,6 +161,12 @@ export default function Preferences() {
                     </div>
                 </SelectableCard>
             </div>
+            <h3 className="mt-12">Dotfiles <PillLabel type="warn" className="font-semibold mt-2 py-0.5 px-2 self-center">Beta</PillLabel></h3>
+            <p className="text-base text-gray-500 dark:text-gray-400">Customise every workspace using dotfiles. Add a repo below which gets cloned and installed during workspace startup.</p>
+            <div className="mt-4">
+                <h4>Repo</h4>
+                <input type="text" value={dotfileRepo} onChange={(e) => actuallySetDotfileRepo(e.target.value)} className="w-full" />
```

Comment:
**suggestion**: Instead of using the section subtitle to describe this input we could use the already existing pattern of the input help text to make it clear to the user what is expected inside the input.